mne/viz/topomap.py

```diff
@@ -1630,8 +1683,11 @@ def plot_evoked_topomap(evoked, times="auto", ch_type=None, layout=None,
                          'Check your input.')
 
     data *= scaling
-    if merge_grads:
+    if merge_grads and ch_type == 'grad':
         data = _merge_grad_data(data)
+    elif merge_grads and ch_type in ['hbo', 'hbr', 'fnirs_raw', 'fnirs_od']:
```

Comment:
I would rename `merge_grads` to `merge_channels` everywhere

Reply:
I have changed it within the range of code that I have modified. Or do you think I should change this globally?

Reply:
I would just change it locally and eventually we can change it to be more global, maybe by putting it in `mne/io/pick.py` and importing it here and elsewhere